src/pages/messages/components/TicketListItem.tsx

```diff
@@ -43,8 +44,20 @@ export const TicketListItem = ({ ticket }: TicketListItemProps) => {
       reference: { publicationInstance: { type: publicationInstance?.type ?? '' } },
     },
   } as Registration;
-  const msAge = new Date().getTime() - new Date(ticket.modifiedDate).getTime();
-  const daysAge = Math.ceil(msAge / 86_400_000); // 1000 * 60 * 60 * 24 = 86_400_000 ms in one day
+
+  const getTodayDateString = () => {
```

Comment:
Trengs strengverdien til nå-dato til noe? Ser ut som et unødvendig ekstra steg med mye kompleksitet nå. Hva med å bare opprette nå-dato direkte i getTimePeriodString() i stedet? Ser ut som den bare trenger å ta inn en dato som argument, og så kan den finne differansen mellom input-dato og nå-dato (som den kan lage selv)

Også: Om vi har behov for strengversjonen (ISO) av en dato burde new Date().toISOString() brukes, både for å unngå egne bugs, og for å beholde info om tidssone etc